src/Sia.Gateway/Initialization/Startup/Startup.cs

```diff
@@ -26,6 +30,34 @@ public Startup(IHostingEnvironment env)
             }
             _configuration = builder.Build();
             _gatewayConfiguration = _configuration.Get<GatewayConfiguration>();
+
+            if (env.IsStaging())
+            {
+                _gatewayConfiguration.GatewayDatabaseConnectionString = _configuration.GetConnectionString(STAGING_DB_CONNECTION_STRING_KEY);
+            }
+            else if (env.IsEnvironment(KUBERNETES_STACK))
+            {
```

Comment:
This would likely be easier to test if it were extracted to its own method.

Reply:
I am still yet to come up with a way to unit test Config processing code.